README.md

```diff
@@ -124,11 +124,11 @@ Currently only SPDX2.2 is supported.
 
 Use the tool to redact any references to files from a given SBOM or set of SBOMs with either of the following commands:
 
-```
+```shell
 sbom-tool redact -sd <directory containing SBOMs to redact> -o <output path>
 ```
 
-```
+```shell
```

Comment:
In this case, we want the < and > to be placeholders, not literals, so we want the generic formatting